src/angular/templates/templateParser.ts

```diff
@@ -69,12 +75,12 @@ export const parseTemplate = (template: string, directives: DirectiveDeclaration
 
   // Make sure it works with 2.2.x & 2.3.x
   const summaryKind = ((compiler as any).CompileSummaryKind || {}).Template;
-  const templateMetadata: compiler.CompileTemplateMetadata = {
+  let templateMetadata: any = {
```

Comment:
Agree. Trying to reduce it to minimum but the constant changes of the Angular's compiler API makes it impossible.